core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3280,6 +3280,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_METADATA_SYNC_AVOID_SAME_PATH_CONCURRENT_SYNC =
+      booleanBuilder(Name.MASTER_METADATA_SYNC_AVOID_SAME_PATH_CONCURRENT_SYNC)
```

Comment:
Such a property exists merely because we are paranoid and want to be able to switch it off when there's a bug. So there's a field called `setIsHidden(true)` so this property is only known internally and not shown on the doc page.

Reply:
i think enabling this feature will cause some side effects so lets expose this for now